operator/main.go

```diff
@@ -288,6 +288,9 @@ func onOperatorStartLeading(ctx context.Context) {
 	var (
 		nodeManager *allocator.NodeEventHandler
 	)
+
+	log.WithField("mode", option.Config.IPAM).Info("Initializing IPAM")
```

Comment:
We should consider adding this to the list of established fields, since it's an `Info` msg.